plugins/asciidoc/asciidoc.py

```diff
@@ -34,6 +34,11 @@
 import os
 import subprocess
 
+# for non-ascii encoding
+import sys
+reload(sys)
+sys.setdefaultencoding('utf-8')
+
```

Comment:
the problem with this is that the asciimodule is an external plugin so if we update it this change will go away. can you try adding:

```py
## -*- coding: utf-8 -*-
```

as the first line of any file with this problem (better even in all the content files to avoid similar issues in the future)

we had this kind of issue often with blogofile and adding that header fixed it